### PR TITLE
Clarify behavior of license expiration

### PR DESCRIPTION
**Pull request by JakeSCahill**
## Description

Review deadline: 2 December

Addresses recent feedback on the licensing doc:

> We should clarify. This table is what happens automatically upon expiry, but there are also more restrictions  after expiry where you can't enable any enterprise feature.

> Also we need to add that Console (on expiry) forwards you to a landing page if CONSOLE RBAC or CONSOLE SSO (not cluster RBAC) is enabled.

## Page previews

https://deploy-preview-878--redpanda-docs-preview.netlify.app/24.3/get-started/licensing/overview/#self-managed

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Review by michael-redpanda — APPROVED** [review]

**Review by oleiman — APPROVED** [review]
lgtm

File: modules/get-started/pages/licensing/monitor-license-status.adoc [hunk 1]
```diff
@@ -142,7 +142,7 @@ Redpanda Console displays warnings in the following scenarios:
 +
 [,json]
 ----
-{"level":"info","msg":"successfully loaded Redpanda Enterprise license","license_org":"redpanda","license_type":"enterprise","expires_at":"Oct 12 2024"}
+{"level":"info","msg":"successfully loaded Redpanda enterprise license","license_org":"redpanda","license_type":"enterprise","expires_at":"Oct 12 2024"}
```
Comment (JakeSCahill) [inline modules/get-started/pages/licensing/monitor-license-status.adoc]:
```suggestion
{"level":"info","msg":"successfully loaded Redpanda Enterprise license","license_org":"redpanda","license_type":"enterprise","expires_at":"Oct 12 2024"}
```

File: modules/get-started/pages/licensing/overview.adoc [hunk 2]
```diff
@@ -67,19 +79,19 @@ The following table lists the available Enterprise features for Redpanda, along
 
 | xref:manage:security/fips-compliance.adoc[FIPS Compliance]
 | Enables compliance with FIPS security standards for cryptography.
-| None
+| No change
```
Comment (JakeSCahill) [inline modules/get-started/pages/licensing/overview.adoc]:
```suggestion
| No change.
```

File: modules/get-started/pages/licensing/overview.adoc [hunk 3]
```diff
@@ -67,19 +79,19 @@ The following table lists the available Enterprise features for Redpanda, along
...
 
 | xref:manage:security/authentication.adoc#kerberos[Kerberos authentication]
 | Provides secure Kerberos-based authentication.
-| None
+| No change
```
Comment (JakeSCahill) [inline modules/get-started/pages/licensing/overview.adoc]:
```suggestion
| No change.
```

File: modules/get-started/pages/licensing/overview.adoc [hunk 4]
```diff
@@ -67,19 +79,19 @@ The following table lists the available Enterprise features for Redpanda, along
...
 
 | xref:manage:security/authentication.adoc#kerberos[Leader Pinning]
 | Specifies the set of availability zones where the leaders of partitions of a given topic should be located.
 | Leader Pinning is disabled on all topics.
 
 | xref:manage:security/authentication.adoc#oidc[OAUTHBEARER/OIDC authentication]
 | Allows for OAUTHBEARER and OpenID Connect (OIDC) authentication.
-| None
+| No change
```
Comment (JakeSCahill) [inline modules/get-started/pages/licensing/overview.adoc]:
```suggestion
| No change.
```

File: modules/get-started/pages/licensing/overview.adoc [hunk 5]
```diff
@@ -67,19 +79,19 @@ The following table lists the available Enterprise features for Redpanda, along
 
 | xref:manage:security/fips-compliance.adoc[FIPS Compliance]
 | Enables compliance with FIPS security standards for cryptography.
-| None
+| No change.
 
 | xref:manage:security/authentication.adoc#kerberos[Kerberos authentication]
```
Comment (asimms41) [inline modules/get-started/pages/licensing/overview.adoc]: Just a style question. Should all links be title case, there seems to be a mix.

**Review by asimms41 — APPROVED** [review]
LGTM to a style question.

**Pull request merged**